usaspending_api/disaster/tests/integration/test_federal_account_total.py

```diff
@@ -25,7 +25,7 @@ def test_federal_account_success(client, generic_account_data, monkeypatch, help
                 }
             ],
             "code": "000-0000",
-            "award_count": None,
```

Comment:
All responses for "total" spending should be returning None

Reply:
Ah.  Ok.  Now your Slack comment makes more sense.  Got it.